### PR TITLE
Adding extra statement in PR template for e2e

### PR DESCRIPTION
**Pull request by vinuthakaranth**
## Description

Adding extra statement in PR template to ensure e2e tests are added if PR has UI changes.

## Checklist

- [ ] I have added screenshots above for all UI changes.
- [ ] Documentation was updated if it was needed.
- [ ] New tests were added or changes were manually verified.

### CODE REVIEW

**Review by gaugup — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -10,5 +10,6 @@
 <!--- Make sure to satisfy all the criteria listed below. -->
 
 - [ ] I have added screenshots above for all UI changes.
+- [ ] I have added e2e tests for all UI changes.
```
[the remark this answers is not in the document]
Comment (vinuthakaranth) [reply .github/PULL_REQUEST_TEMPLATE.md:13]: I tried to that but I wanted it to be more explicit. Please suggest the statement. I am out of ideas :D

[the remark this answers is not in the document]
Comment (vinuthakaranth) [reply .github/PULL_REQUEST_TEMPLATE.md:13]: I can remove last statement if it means the same as this statement.

[the remark this answers is not in the document]
Comment (vinuthakaranth) [reply .github/PULL_REQUEST_TEMPLATE.md:13]: removed last statement.

[the remark this answers is not in the document]
Comment (romanlutz) [reply .github/PULL_REQUEST_TEMPLATE.md:13]: OK that works for me.

**Review by romanlutz — APPROVED** [review]
BTW feel free to merge this into main since it has no effect on code that we may need to ship urgently.

**Pull request merged**